src/js/components/DataTable/__tests__/__snapshots__/DataTable-test.js.snap

```diff
@@ -5962,6 +5962,11 @@ exports[`DataTable search 2`] = `
   outline: none;
 }
 
+.c2:-moz-placeholder,
+.c2::-moz-placeholder {
+  opacity: 1;
+}
+
```

Comment:
All the snapshot changes with this CSS change make sense to me, though we would need to communicate it well on the release notes to call out this is not a backward compatibility issue but an educated move by grommet for cross-browser consistency. Please come up with 1-2 lines for the release notes that explain the impact this PR might have on existing applications to call it out in advance.

Reply:
Sounds good, I will do that!

Reply:
In order to provide cross-browser consistency, an opacity of 1 has been added to Firefox’s rendering of placeholder text to override its default placeholder opacity of 0.54, which other browsers do not apply. This change ensures the visual appearance of placeholder text is consistent across browsers.